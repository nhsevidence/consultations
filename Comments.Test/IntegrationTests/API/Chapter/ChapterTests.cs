using System;
using System.Linq;
using System.Net;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Comments.Test.Infrastructure;
using Comments.ViewModels;
using Newtonsoft.Json;
using Shouldly;
using Xunit;

namespace Comments.Test.IntegrationTests.API.Chapter
{
    public class ChapterTests : TestBase
    {
        [Fact]
        public async Task Get_Chapter_Feed_ReturnsEmptyFeed()
        {
            //Arrange (in the base constructor for this one.)

            // Act
            var response = await _client.GetAsync("/consultations/api/Chapter?consultationId=1&documentId=2&chapterSlug=introduction");
            response.EnsureSuccessStatusCode();
            var responseString = await response.Content.ReadAsStringAsync();

            // Assert
            responseString.ShouldMatchApproved();
        }


    }
}
