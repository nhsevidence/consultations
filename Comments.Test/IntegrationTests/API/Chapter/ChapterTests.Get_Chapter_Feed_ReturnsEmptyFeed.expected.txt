{
    "content":null,
    "sections":null,
    "slug":null,
    "title":"For consultation comments"
}